src/controllers/album.js

```diff
@@ -0,0 +1,92 @@
+const db = require("../db/index");
+
+const createAlbum = async (req, res) => {
+  const { id: artistId } = req.params;
+  const { name, year } = req.body;
+
+  try {
+    const {
+      rows: [album],
+    } = await db.query(
+      `INSERT INTO Albums (name, year, artistId) VALUES ($1, $2, $3) RETURNING *`,
+      [name, year, artistId]
+    );
+    res.status(201).json(album);
+  } catch (err) {
+    res.status(500).json(err.message);
+  }
+};
+
+const readAlbums = async (req, res) => {
+  try {
+    const response = await db.query("SELECT * FROM Albums");
+    const albums = response.rows;
+    res.status(200).json(albums);
+  } catch (err) {
+    res.status(500).json(err.message);
+  }
+};
+
+const getAlbumById = async (req, res) => {
+  try {
+    const { id } = req.params;
```

Comment:
I would move this line before the `try`, as this happens before any asynchronous code i.e.
```
const { id } = req.params;

try {
```